<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>telemetryDialog</class>
 <widget class="QDialog" name="telemetryDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>763</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>741</width>
     <height>461</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="Telemetry">
    <attribute name="title">
     <string>Telemetry</string>
    </attribute>
    <widget class="QDial" name="WSdial">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>181</width>
       <height>181</height>
      </rect>
     </property>
     <property name="maximum">
      <number>1200</number>
     </property>
     <property name="singleStep">
      <number>10</number>
     </property>
     <property name="notchTarget">
      <double>10.000000000000000</double>
     </property>
     <property name="notchesVisible">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="WSvalue">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>WS How High</string>
     </property>
    </widget>
    <widget class="QPushButton" name="startButton">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>400</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="exitButton">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>400</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Module">
    <attribute name="title">
     <string>Module</string>
    </attribute>
    <widget class="QPushButton" name="startButtonModule">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>400</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="exitButtonModule">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>400</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>190</y>
       <width>341</width>
       <height>201</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>321</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QSpinBox" name="A11AlarmSB">
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QComboBox" name="A11GtLtCB">
         <item>
          <property name="text">
           <string>&gt;</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>&lt;</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_7">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>A1(1)</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QComboBox" name="A11LevelCB">
         <item>
          <property name="text">
           <string>Disabled</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Yellow</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Orange</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Red</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>321</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="1">
        <widget class="QSpinBox" name="A12AlarmSB">
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QComboBox" name="A12GtLtCB">
         <item>
          <property name="text">
           <string>&gt;</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>&lt;</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_8">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>A1(2)</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QComboBox" name="A12LevelCB">
         <item>
          <property name="text">
           <string>Disabled</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Yellow</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Orange</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Red</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>321</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="1">
        <widget class="QSpinBox" name="A21AlarmSB">
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QComboBox" name="A21GtLtCB">
         <item>
          <property name="text">
           <string>&gt;</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>&lt;</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_11">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>A2(1)</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QComboBox" name="A21LevelCB">
         <item>
          <property name="text">
           <string>Disabled</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Yellow</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Orange</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Red</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>321</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="1">
        <widget class="QSpinBox" name="A22AlarmSB">
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QComboBox" name="A22GtLtCB">
         <item>
          <property name="text">
           <string>&gt;</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>&lt;</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_12">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>A2(2)</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QComboBox" name="A22LevelCB">
         <item>
          <property name="text">
           <string>Disabled</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Yellow</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Orange</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Red</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>321</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="1">
        <widget class="QSpinBox" name="RssiAlarmSB">
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_9">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Rx RSSI</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QComboBox" name="RssiLevelCB">
         <item>
          <property name="text">
           <string>Disabled</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Yellow</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Orange</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Red</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="ReadAlarmsButton">
         <property name="text">
          <string>Read Alarms</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>321</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="1">
        <widget class="QSpinBox" name="TssiAlarmSB">
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_10">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Tx RSSI</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QComboBox" name="TssiLevelCB">
         <item>
          <property name="text">
           <string>Disabled</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Yellow</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Orange</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Red</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="SetAlarmsButton">
         <property name="text">
          <string>Set Alarms</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>20</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Alt</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>50</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Alt(d)</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>251</width>
       <height>71</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>40</y>
        <width>46</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>RxRSSI</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="A1SB">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="RSSI_SB">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>10</y>
        <width>46</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>A1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>40</y>
        <width>46</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>TxRSSI</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="TSSI_SB">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>40</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="A2SB">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>10</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>10</y>
        <width>46</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>A2</string>
      </property>
     </widget>
    </widget>
    <widget class="QLineEdit" name="AltDisp">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>20</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="AltDdisp">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>50</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="startButtonLogging">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>10</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Log</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="SportUart">
    <attribute name="title">
     <string>SportUart</string>
    </attribute>
    <widget class="QPushButton" name="startButtonUart">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>80</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>170</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="display">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>401</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="BaudrateCB">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>300</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4800</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9600</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>41</y>
       <width>46</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Baudrate</string>
     </property>
    </widget>
    <widget class="QComboBox" name="CorrectCB">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>No Correction</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Self Correction</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="DirectionCB">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>100</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Uni-Direction</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Bi-Direction</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="SetBaud">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>40</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Set</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SetCorrect">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>70</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Set</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SetDirect">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>100</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Set</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QComboBox" name="TelPortCB">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Com Port</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
